tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpClientTestSupport.java

```diff
@@ -256,21 +262,46 @@ protected void configureAddressPolicy(ActiveMQServer server) {
 
    protected void createAddressAndQueues(ActiveMQServer server) throws Exception {
       // Default Queue
-      server.addAddressInfo(new AddressInfo(SimpleString.toSimpleString(getQueueName()), RoutingType.ANYCAST));
-      server.createQueue(new QueueConfiguration(getQueueName()).setRoutingType(RoutingType.ANYCAST));
+      try {
+         server.addAddressInfo(new AddressInfo(SimpleString.toSimpleString(getQueueName()), RoutingType.ANYCAST));
+      } catch (Throwable ignored) {
+      }
```

Comment:
Why is this needed? Swallowing test setup exceptions generally just to cater for a specific test seems like a bad idea. 

If its something like 'address already exists' the test should likely be overriding the method and changing it to accommodate its test-specific needs (e.g get and clear a previously-set reference to some test-specific / startup-specific behaviour to execute).

Reply:
Chained brokers will create the destinations on all the servers, however the mirror will also create the destinations.


I will rather change the test.. this was just a hack for me to debug, I don't intend to leave this change on.

Reply:
I didn't intend to keep this change. it was just a hack for my investigation.

The git commit is still Work in Progress but I removed it just to avoid confusion.

I should have kept another branch with my backup away from the PR. sorry for the confusion.

Reply:
I fixed the test accordingly. Creates are now propagated accordingly. I actually changed the test to validate that a destination created on the first server ended up on the third as expected.